eng/build.sh

```diff
@@ -21,8 +21,10 @@ usage()
   echo "  --subsetCategory           Build a subsetCategory, print available subsetCategories with -subset help"
   echo "  --os                       Build operating system: Windows_NT or Unix"
   echo "  --arch                     Build platform: x86, x64, arm or arm64"
-  echo "  --configuration <value>    Build configuration: Debug or Release (short: -c)"
-  echo "  --verbosity <value>        MSBuild verbosity: q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic] (short: -v)"
+  echo "  --configuration            Build configuration: Debug, Release or [CoreCLR]Checked (short: -c)"
+  echo "  --runtimeconfiguration     Runtime build configuration: Debug, Release or [CoreCLR]Checked"
+  echo "  --librariesconfiguration   Libraries build configuration: Debug or Release"
```

Comment:
(Nit) Capitalize `c` in the above two lines to match ps `librariesConfiguration` and earlier arg docs in this file like `--subsetCategory`.